src/forms/Helper.ts

```diff
@@ -1,13 +1,27 @@
-import { ValidationMode } from '@jsonforms/core';
+import { Layout, UISchemaElement, ValidationMode } from '@jsonforms/core';
 import { materialRenderers } from '@jsonforms/material-renderers';
+import {
+    CollapsibleGroup,
+    collapsibleGroupTester,
+} from './renderers/CollapsibleGroup';
+import { NullType, nullTypeTester } from './renderers/NullType';
+import { generateCustomUISchema } from './uischema';
 
 export const defaultOptions = {
     restrict: true,
     showUnfocusedDescription: true,
 };
 
-export const defaultRenderers = [...materialRenderers];
+export const defaultRenderers = [
+    ...materialRenderers,
+    { tester: nullTypeTester, renderer: NullType },
+    { tester: collapsibleGroupTester, renderer: CollapsibleGroup },
+];
 
 export const showValidation = (val: any): ValidationMode => {
     return val ? 'ValidateAndShow' : 'ValidateAndHide';
 };
+
+export const generateUISchema = (jsonschema: any): UISchemaElement => {
+    return generateCustomUISchema(jsonschema, 'VerticalLayout') as Layout;
+};
```

Comment:
I added an extra layer for this call so if we want to we can mess with the UI Schema. Initially my plan was to put all the collapsible groups at the end but decided against it. However - I think having a place to "play" with the response is valuable.

It also keeps the generate more pure so we can diff it against the original copy.